sci-misc/boinc/boinc-7.6.33-r1.ebuild

```diff
@@ -99,7 +114,13 @@ src_install() {
 	keepdir /var/lib/${PN}
 
 	if use X; then
-		newicon "${S}"/packages/generic/sea/${PN}mgr.48x48.png ${PN}.png
+		# Create new icons. bug 593362
+		local s
+		local SIZES=(256 192 128 96 72 64 48 36 32 24 22 16)
```

Comment:
compress this into one line:
`local s SIZES=( 16 22 24 32 36 48 64 72 96 128 192 256 )`
(notice that I sorted it by size)